mir-module/src/main/resources/xsl/modsdetails-external.xsl

```diff
@@ -503,7 +504,7 @@
                   <xsl:sort select="@type" />
                   <xsl:sort select="@id" />
                   <xsl:if test="@permission='true'">
-                    <li>
+                    <li class="mir-action-item mir-action-item-register-doi">
```

Comment:
mir-action-item-register-doi should be more generic, because or there should be one variant for for pi and one for the specific type.
mir-action-item-register-pi and mir-action-item-register-{@type}